scripts/helper/create-cluster.sh

```diff
@@ -80,9 +80,15 @@ case "$OCP_VERSION" in
 		RHCOS_VERSION="4.11"
```

Comment:
It should be: 
4.12)
                OCP_RELEASE="4.12.3"
                RHCOS_VERSION="4.12"
                if [ -z "$RHCOS_RELEASE" ]; then
                        RHCOS_RELEASE="4.12.2"                   # Latest release of RHCOS 4.12 at this time
                fi
                RHCOS_SUFFIX="-$RHCOS_RELEASE"
                ;;

Reply:
Okie